### PR TITLE
updated the bg color

### PR DESCRIPTION
**Pull request by azdobatch12**
updated the bg color to blue

### CODE REVIEW

File: columns.html [hunk 1]
```diff
@@ -13,9 +13,9 @@
     <title>Layouts</title>
   </head>
 
-  <body class="bg-dark">
+  <body class="bg-info">
```
Comment (azdevops10) [inline columns.html]: please change it to red instead of blue (as per the requirement)

File: columns.html [hunk 2]
```diff
@@ -13,9 +13,9 @@
...
     <header id="page-hero" class="site-header d-flex flex-column align-content-between">
-      <nav class="site-nav family-sans navbar navbar-expand-md bg-dark navbar-dark">
+      <nav class="site-nav family-sans navbar navbar-expand-md bg-info navbar-dark">
```
Comment (azdevops10) [inline columns.html]: apply the same for all

File: columns.html (hunk 1 above)
Comment (azdobatch12) [reply columns.html]: updated accordingly.

File: columns.html (hunk 2 above)
Comment (azdobatch12) [reply columns.html]: updated acc...

**Review by azdevops10 — APPROVED** [review]

**Review by azdevops08 — APPROVED** [review]

**Pull request merged**